RoboFile.php

```diff
@@ -529,6 +567,23 @@ public function generateTask($className, $wrapperClassName = "")
         $template = file_get_contents(__DIR__ . "/GeneratedWrapper.tmpl");
         $template = str_replace(array_keys($replacements), array_values($replacements), $template);
 
+        // Returning a string will cause Robo to print it and then
+        // exit with a "no error" (status code 0) result.
         return $template;
     }
 }
+
+class ExceptionTask extends \Robo\Task\BaseTask
```

Comment:
RoboFile is just an example file, so perhaps it is okay to put this example task here?  I don't really want to put it anywhere in 'Tasks'.  The danger of doing this, I suppose, is that maybe some folks might start implementing their own custom tasks right in their RoboFiles. This will give them deprecation errors when their task attempts to print, of course. Not sure what to do here; we could just delete this example, but I think it's kind of nice to have.